frontend/src/models/cohortsModel.ts

```diff
@@ -40,6 +40,18 @@ export const DEFAULT_COHORT_FILTERS: CohortFilters = {
     page: 1,
 }
 
+export function generateUUID(): string {
```

Comment:
Thanks @haacked for the suggestion. Do you think we should move this function into `frontend/src/lib/utils`?

Reply:
Yeah, that makes sense. Then we can delete the one in `frontend/src/scenes/feature-flags/FeatureFlagReleaseConditionsLogic.ts`